monoio/tests/tcp_connect.rs

```diff
@@ -118,11 +126,33 @@ async fn connect_invalid_dst() {
     assert!(TcpStream::connect("127.0.0.1:1").await.is_err());
 }
 
+static TEST_SERVER_PORT: LazyLock<u16> = LazyLock::new(|| {
```

Comment:
What's the LazyLock is for? If the propose is for keep it open, maybe we can just leak it(`Box::leak(Box::new(...))` or with `ManuallyDrop`) since they are test? :)